test/meson.build

```diff
@@ -77,3 +86,13 @@ test(
   env : ['SHARNESS_BUILD_DIRECTORY=' + meson.build_root()],
   workdir : meson.current_source_dir(),
 )
+
+install_data(
+  meson.build_root() / 'config.h',
+  install_dir: get_option('libdir') / 'rauc' / 'tests',
+)
+
+install_subdir(
+  meson.source_root() / 'test',
```

Comment:
This installs way too many files. Also test results, etc. Should be more explicit.

Reply:
So far, we've not split out the subset of files needed by the test (e.g. from the openssl-* directories), as they were available anyway.

Reply:
I have also seen this and tried to move all assets into subdirectories. But this will require a lot of changes in the `rauc.t` script and the unit tests. The additional files installed are not too many (the unit-test c-files and some build support files).

This would be doable but i wanted a first feedback.